cards.c

```diff
@@ -0,0 +1,39 @@
+/* Mariposa Lee, drawing 5 random cards, incomplete. I started over twice, but at the end I couldn't figure it out. Tried in enums first, but then got confused, so I tried to do it with strings...*/
```

Comment:
You had a lot of simple syntax mistakes that could have easily been avoided. Make sure to pay attention to every single thing you type when you're coding. You also should have submitted this code with thorough comments since it isn't working.